### DISCUSSION TITLE
Error in UseMethod("accuracy")

### DISCUSSION THREAD

**Discussion by galuhchynintya** [teamalgoritma/community#906] [Q&A] [answered]
Dear tim algoritma,

Saya mengalami kendala saat melakukan model evaluasi holt winter, berikut screenshot error nya
![Screenshot 2024-08-18 205050](https://github.com/user-attachments/assets/5be819e7-0f64-428b-b8d4-e30e1b7290b0)
Error in UseMethod("accuracy") : 
  no applicable method for 'accuracy' applied to an object of class "ts"

untuk data test bukankah sudah benar kalau bentuk nya dalam ts object?
data dijadikan dalam bentuk ts object terlebih dahulu baru dilakukan cross validation.

Mohon pencerahannya, terima kasih

**Comment by IrfanChairurrachman** [comment, marked as answer]
Dear Bu @galuhchynintya ,

Sepertinya terdapat konflik ketika load fungsi `accuracy()`, yang kita inginkan adalah `accuracy` dari library `forecast` namun yang ter-load adalah accuracy dari library lain. Untuk mengatasi hal ini, silahkan tambahkan `nama_library::` di depan fungsi, seperti berikut:
`forecast::accuracy(forecast_hw$mean, room_test)`

Semoga informasi ini dapat membantu. Bila ternyata informasi ini dapat menjawab pertanyaan Anda, Anda dapat “Mark as answer” jawaban ini.

**Comment by galuhchynintya** [comment]
Dear mas Irfan,
Iya sepertinya betul yang disampaikan mas Irfan. Setelah menambahkan nama library di depan fungsinya, syntax tsb sudah bisa dijalankan. Terima kasih
